src/lib/cookie/cookie.js

```diff
@@ -98,6 +119,16 @@ function encodeVendorConsentData(vendorData) {
 	return noRangesData.length < rangesData.length ? noRangesData : rangesData;
 }
 
+
+function decodeBitsToIds(bitString) {
+	return bitString.split('').reduce((acc, bit, index) => {
+		if (bit === '1') {
+			acc.add(index + 1);
+		}
+		return acc;
+	}, new Set());
+}
+
 function decodeVendorConsentData(cookieValue) {
```

Comment:
This function is no longer used

Reply:
I've moved it to separate file which is bundled only for docs.